### ISSUE TITLE
Gujarati, Hindi and Sanskrit Language OCR not working

### ISSUE THREAD

**Issue by vikithakar** [ciur/papermerge#583]
![Screenshot from 2024-01-22 18-03-00](https://github.com/ciur/papermerge/assets/1167994/bf37ee5d-f18e-4996-9e88-eb92cfd660d6)

### Description of Issue
After building Papermerge with `Gujarati`, `Hindi` and `Sanskrit` Language support, when you upload and run OCR on files, it churns out OCR text which is not correct. I think the tesseract-ocr is consistent with the text output it gives for the file, but it seems like papermerge does not have fonts or Character Sets to display the translations in the OCR text language. 

### Build Details

`Dockerfile` to add tesseract-ocr to papermerge 
```
FROM papermerge/papermerge:3.0.2
RUN apt install tesseract-ocr-hin tesseract-ocr-guj tesseract-ocr-san -y
```

**Info:**
 - Papermerge Version 3.0.2

**Comment by ciur** [comment]
Thank you for reporting the issue!

**Comment by ciur** [comment]
In order to make this work, I need to include `Gujarati`, `Hindi` and `Sanskrit` codes [here](https://github.com/papermerge/papermerge-core/blob/master/papermerge/core/schemas/tasks.py#L6) and [here](https://github.com/papermerge/papermerge-core/blob/master/ui/src/cconstants.ts#L10). For the second list, I need
respective language written in original language; for example "fra" French is "Français".
Could you please provide original writing of the language name for  `Gujarati`, `Hindi` and `Sanskrit` ?

- guj:  ?
- hin: ?
- san: ?

**Comment by vikithakar** [comment]
@ciur 
Original Language Name 

- `guj` in Gujarati is ગુજરાતી
- `hin` in Hindi is हिंदी
- `san` in Sanskrit is संस्कृत

**Comment by ciur** [comment]
[PR](https://github.com/papermerge/papermerge-core/pull/323) for adding above mentioned languages.

Change will be available in 3.0.3 release

Note that you will need to build your image as before. However, when you will start papermerge don't forget to add [PAPERMERGE__OCR__DEFAULT_LANGUAGE](https://docs.papermerge.io/3.0/user/ocr/?h=default#default-ocr-language) variable so that when you import docs they will be OCRed in "default OCR" language.

**Comment by ciur** [comment]
Here is screenshot with working app (as mentioned above will be part of 3.0.3):

![papermerge-with-hindi-text](https://github.com/ciur/papermerge/assets/24827601/fe79438a-9223-45a4-8adf-7b88899869fc)
